input/fsh/profiles/Encounter_Maternity.fsh

```diff
@@ -46,7 +46,7 @@ Description: "This Encounter profile represents the mother's maternity encounter
 * location MS
   * ^short = "Information about the location of delivery"
   * ^definition = "Information about the location of devivery"
-  * location only Reference(USCoreLocation)
+  * location only Reference(LocationBFDR)
   * location MS
     * ^short = "Facility name (if not institution, give street and number)"
     * ^definition = "Facility name (if not institution, give street and number)"
```

Comment:
"HOSP_D" maps to location.location.name - not currently in differential

Reply:
Several other fields should have comments to show in the differential:
-	Encounter maternity: location.location.address.extension[XXXX] + maps to various address-related codes 
-	Encounter maternity: location.location.address.line/postalCode/district/city map to various location codes
-	Encounter maternity: “CNTYO” mapped to location.location.address.district.extension[countyCode] 
-	Encounter maternity: “HOSPFROM” mapped to hospitalization.origin.name

Reply:
These map to LocationBFDR